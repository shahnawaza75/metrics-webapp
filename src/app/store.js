import { configureStore } from '@reduxjs/toolkit';
import gameReducer from '../redux/gameSlice';

const store = configureStore({
  reducer: {
    game: gameReducer,
  },
});

export default store;
