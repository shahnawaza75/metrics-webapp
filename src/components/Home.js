import React from 'react'
import Header from './Header'
import Container from './Container'

function Home() {
    return (
        <div>

            <Header />
            <Container />

        </div>

    )
}

export default Home

