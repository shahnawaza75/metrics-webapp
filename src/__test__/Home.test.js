import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../app/store';
import Home from '../components/Home';

const HomeStore = () => (
  <Provider store={store}>
    <Home />
  </Provider>
);

describe('Test Home Components', () => {
  it('Output', () => {
    const home = renderer.create(
      <HomeStore />,
    ).toJSON();

    expect(home).toMatchSnapshot();
  });
});
