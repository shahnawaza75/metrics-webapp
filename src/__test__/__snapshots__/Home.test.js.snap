// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home Components Output 1`] = `
<div>
  <div
    className="flex bg-pink-100 flex-col text-white"
  >
    <div
      className="flex items-center justify-center space-x-2 h-32"
    >
      <img
        alt="crypto"
        className="logoun w-1/2"
        src="cryptocurrencyimage.jpeg"
      />
      <div>
        <h1
          className="font-extrabold text-2xl"
        >
          Crypto Exhange
        </h1>
        <p
          className="font-semibold text-lg"
        >
          0
           
          crypto
        </p>
      </div>
    </div>
    <div
      className="text-start pl-5 bg-pink-200 text-xl font-medium"
    >
      Stats
    </div>
    <ul
      className="list-items grid grid-cols-2"
    />
  </div>
</div>
`;
